tests/sysauth/sssd.pm

```diff
@@ -29,10 +29,12 @@ sub run {
     if (is_sle) {
         assert_script_run 'source /etc/os-release';
         if (is_sle '>=15') {
-            if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
-                record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';
-                assert_script_run 'SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300;
-            }
+            my $version //= '${VERSION_ID}';
```

Comment:
That shouldn't have `//=`. It will work with it anyway, but, someone who reads this can think that we expect those variables to be initialized befire.

```suggestion
            my $version = '${VERSION_ID}';
```

Reply:
okay , update now